context/app/static/js/pages/search/DevSearch.jsx

```diff
@@ -88,10 +88,8 @@ function DevSearch() {
         checkboxFilter('no_metadata', 'No metadata?', BoolMustNot(ExistsQuery('metadata.metadata'))),
         checkboxFilter('has_files', 'Has files?', ExistsQuery('files')),
         checkboxFilter('no_files', 'No files?', BoolMustNot(ExistsQuery('files'))),
-        checkboxFilter('has_rui_sample', 'Spatial Sample?', ExistsQuery('rui_location')),
-        checkboxFilter('no_rui_sample', 'Not Spatial Sample?', BoolMustNot(ExistsQuery('rui_location'))),
-        checkboxFilter('has_rui_dataset', 'Spatial Dataset?', ExistsQuery('ancestors.rui_location')),
-        checkboxFilter('no_rui_dataset', 'Not Spatial Dataset?', BoolMustNot(ExistsQuery('ancestors.rui_location'))),
+        checkboxFilter('is_spatial', 'Spatial?', ExistsQuery('rui_location')),
+        checkboxFilter('not_spatial', 'Not Spatial?', BoolMustNot(ExistsQuery('rui_location'))),
```

Comment:
Like above, I think this won't work for datasets unless there was some change I'm not aware of?

Reply:
This is now supported with the change I linked earlier.